### PR TITLE
updating storybook config

### PR DESCRIPTION
**Pull request by iruzevic**
I have changes stuff a little bit. In general there are a lot of changes but only because I have moved blocks from `blocks>init>blocks` to `blocks>init>src>blocks`. This was done so se can have consistant folder structure to a final project. @IvanGrginovInf there will be changes in the rename script just to fix this folder changes

Also there are some changes on storybook config so it can be included in to each project with few easy steps.

### CODE REVIEW

File: .storybook/parts/default-categories.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/**
+ * Manualy populate categories for blocks. This is generated in the PHP part of the real project.
+ */
+wp.data.dispatch( 'core/blocks').setCategories([
+  {
+    slug: 'eightshift',
+    title: 'Eightshift',
+    'icon': 'admin-settings',
```
Comment (dingo-d) [inline .storybook/parts/default-categories.js]:
```suggestion
    icon: 'admin-settings',
```

This doesn't have to be quoted?

File: .storybook/preview-head.html [hunk 2]
```diff
@@ -1,3 +1,7 @@
 <script>
   window.wp = {};
 </script>
+
+<!-- REMOVE ONCE jQuery is removed from the libs. -->
```
Comment (dingo-d) [inline .storybook/preview-head.html]: Do we have an issue for this? Because _TODO_s are often forgotten in the code.

File: webpack/storybook.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+      },
+    ],
+  });
+
+  /**
+   * Extract css from sass.
+   */
+  config.plugins.push(new MiniCssExtractPlugin());
+
+  // REMOVE ONCE jQuery is removed from the libs.
```
Comment (dingo-d) [inline webpack/storybook.js]: Same as with the previous comment

File: .storybook/preview-head.html (hunk 2 above)
Comment (kmaric69) [reply .storybook/preview-head.html]: yes, there is an open issue for this:
https://github.com/infinum/eightshift-frontend-libs/issues/42#event-2893662111

File: webpack/storybook.js (hunk 3 above)
Comment (kmaric69) [reply webpack/storybook.js]: 🆙

File: .storybook/preview-head.html (hunk 2 above)
Comment (dingo-d) [reply .storybook/preview-head.html]: Ok, then this comment needs to be removed 🙂

**Comment by iruzevic** [comment]
> Will this enable users to locally run storybooks?
> 
> Also, please don't merge this before create a PR fixing the setup script for the new paths.

Yes it will :D

**Comment by iruzevic** [comment]
Fixed PRs

**Pull request merged**